packages/migrations/src/migration.ts

```diff
@@ -284,6 +284,8 @@ export async function runMigrationsAsync(
         exchange: exchange.address,
         // TODO (xianny): figure out how to deploy AssetProxyOwnerContract
```

Comment:
Nit: Can this be removed, or are we still waiting for the migration to be updated for the `ZeroExGovernor`?

Reply:
Probably. This file actually doesn't look very up to date. I haven't been modifying it except to get it to build.